docs/openapi.yaml

```diff
@@ -31,6 +31,61 @@ paths:
       tags:
         - admin
 
+  /admin/announcements:
+    summary: >
+      Create/Edit/Read global announcements meant for every library. \
+      The announcements will show up in the authentication documents.
+    get:
+      summary: Fetch all global announcements
+      tags:
+        - admin
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  settings:
+                    type: object
+                  announcements:
+                    type: object
+    post:
+      summary: Edit the list of global announcements in its entirety.
+      tags:
+        - admin
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                announcements:
+                  type: string
+                  description: JSON stringified list of announcements
+                  example: [{"id": "xxxx-xxx-xxxxx", "message": "An announcement for all to see", start: "1990-01-01"}, {"message": "This is a new announcement without an id yet"}]
```

Comment:
I believe "message" should be "content" in the example.

Reply:
I got my own documentation wrong :no_mouth: